tools/cli-bootstrap/dockerfiles/debian8/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM debian:8.0
```

Comment:
Should these dockerfiles just be removed for now, then? It doesn't even look like most of them will actually build at all since they're all using the same Ubuntu commands.